main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -1591,7 +1591,7 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
 
     def QualifiedDefinition: Rule1[Name.QName] = LowerCaseQName
 
-    def Eff: Rule1[Name.Ident] = LowerCaseName
+    def Pur: Rule1[Name.Ident] = LowerCaseName
```

Comment:
Lets call this one Purity.

Reply:
Since this is actually the name for an effect declaration I'll change it back to `Effect`.